src/services/resource.js

```diff
@@ -226,13 +234,119 @@ module.exports = class resourceHelper {
 		// generic function to merge all the collected data about the resource
 		result = await this.responseBuilder(response, userDetails, orgDetails, additionalResourceInformation)
 		// count of requested for changes resources
-		result.changes_requested_count = distinctInreviewResourceIds.count > 0 ? distinctInreviewResourceIds.count : 0
+		result.changes_requested_count = requestedForChangesResources.count > 0 ? requestedForChangesResources.count : 0
 		return responses.successResponse({
 			statusCode: httpStatusCode.ok,
 			message: 'RESOURCE_LISTED_SUCCESSFULLY',
 			result,
 		})
 	}
+
+	/**
+	 * Return the mapping of resource Id and its final status
+	 * @name finalResourceStatus
+	 * @param {Array} resourceIds - Array of unique resource ids
+	 * @param {Array} organizationIds - Array of unique org ids
+	 * @returns {JSON} - List of resources with final status
+	 */
+	static async finalResourceStatus(resourceIds, organizationIds) {
```

Comment:
@priyanka-TL Check if this and below function is really needed.

Reply:
yes its required, for the scenario which we discussed in morning